Classes/Uri/TyposcriptRenderingUri.php

```diff
@@ -55,7 +55,7 @@ private function parseViewHelperContext(ViewHelperContext $viewHelperContext): v
         $pluginName = $arguments['pluginName'] ?? null;
         $extensionName = $arguments['extensionName'] ?? null;
         $contextRecord = $arguments['contextRecord'];
-        $additionalParams = $arguments['additionalParams'];
+        $additionalParams = $arguments['additionalParams'] ?? [];
```

Comment:
All these arguments are initialized with the respective default values by Fluid, aren't they?

Reply:
when you're using the ViewHelpers yet, but not necessarily when you're using the classes directly

Reply:
H, ok, fine. If you remove the array_key_exists, I'll merge this here.

Reply:
there you go :)